### PR TITLE
[ADD] vtiger_connector_crm: Added v17 migrated module

### PR DESCRIPTION
**Pull request by RohanChristianSCS**

### CODE REVIEW

File: vtiger_connector_crm/__pycache__/__init__.cpython-310.pyc
Comment (chandni-serpentcs) [inline vtiger_connector_crm/__pycache__/__init__.cpython-310.pyc:1]: Please remove .pyc file

File: vtiger_connector_crm/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def action_sync_vtiger(self):
```
Comment (chandni-serpentcs) [inline vtiger_connector_crm/models/res_company.py:13]: Do we need to this method?Because we can directly use "sync_vtiger_crm" from button

Comment (RohanChristianSCS) [reply vtiger_connector_crm/models/res_company.py:13]: Yes, we require this method because it is a common method for scheduler.

**Review by chandni-serpentcs — APPROVED** [review]

**Comment by chandni-serpentcs** [comment]
Ready for merge

**Pull request merged**